utils/companyDataService.tsx

```diff
@@ -1,43 +1,57 @@
-import * as fs from 'fs'
-import path from 'path'
+import fetch from 'node-fetch'
 import { Company, CompanyEmissionsPerYear } from './types'
 
 export class CompanyDataService {
-  companies: Array<Company>
+  companies: Array<Company> = []
 
   constructor() {
-    const companiesDataFilePath = path.resolve('./data/companies/company-data.json')
-    const companiesDataFileContent = fs.readFileSync(companiesDataFilePath, {
-      encoding: 'utf-8',
-    })
-    const jsonData = JSON.parse(companiesDataFileContent)
-
-    this.companies = jsonData
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      .map((data: any) => {
-        const emissionsPerYear = {
-          Scope1n2: data.Scope1n2,
-          Scope3: data.Scope3,
-        } as unknown as CompanyEmissionsPerYear
+    this.loadCompanies() // Ensure data is loaded asynchronously
+  }
+
+  // Load companies data asynchronously
+  private async loadCompanies() {
+    try {
+      const response = await fetch('https://api.klimatkollen.se/api/companies')
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch data from the API')
+      }
+
+      const jsonData = await response.json()
```

Comment:
This is not thorough review, but it looks good from a quick look! :)

We get an error in the vercel build regarding the response type here. We could use the same response type as the one from https://github.com/Klimatbyran/beta/blob/d27544f1aedaa5128eae80052a85a74bdfff06f2/src/data/companies.ts#L4 to fix this issue and allow the build to complete.

```sh

Type error: 'jsonData' is of type 'unknown'.
  20 |       const jsonData = await response.json()
  21 |
> 22 |       this.companies = jsonData.map((data: any) => {
```